### ISSUE TITLE
Selenium node is not showing on the Grid Console

### ISSUE THREAD

**Issue by RayChenLDL** [mhardin/SeleniumGridScaler#25]
Hi,

I am able to start Selenium hub and sent a request HTTP request to spin up 1 EC2 node. I can confirm from AWS website, the EC2 instance (c4.large) had been spin up and running while I checked my selenium hub console, nothing is shown on it.

Anyone has any idea please?

http://HUBIP:4444/grid/admin/AutomationTestRunServlet?uuid=testRun1&threadCount=5&browser=chrome

**Comment by RayChenLDL** [comment]
BTW, after sending the HTTP request, I waited more than a few minutes until the EC2 instance is running (from aws website, the "instance state is running"). Then I checked my selenium hub console. unfortunately, nothing is there.

**Comment by suveerprithipal** [comment]
Yeah, so I'm investigating the same issue - I assume its to do with the fact that we are not registering the nodes on the hub(albeit the author makes no mention of this on the ReadMe.md).
I make this assumption as this is what we would traditionally do on a selenium grid setup. 
I'll keep this post updated should I find a solution. Please can I ask you do the same.

**Comment by suveerprithipal** [comment]
@mhardin @ambirag, could you lend your experience with us as I am unable to find a solution to this. 
Do the AMI's for the nodes require any special attributes like a node.json config attached to them? 

Thanks gents,
SP

**Comment by mhardin** [comment]
@suveerprithipal @RayChenLDL if a node is not connecting to the hub, possible causes could be the wrong IP address for the hub (for the node to connect to) was provided in the config (see readme), or another cause would be if the node does not have network access to that IP.  Either way, if you login to the node, there should be a grid.log log file you can check to see why your node is not connecting.

**Comment by datlq53** [comment]
I'm facing the same issue
I use AMI: ami-00d4e9ff62bc40e03(default  AMI for ubuntu 14.04) for node, Port 4444 on my selenium hub instance is public.
On AWS console, there is one node is running, but nothing on grid console.
Could you please give me a solution.

Thanks!
